src/v2/Apps/Settings/Routes/MyCollection/Components/MyCollectionGridItem.tsx

```diff
@@ -0,0 +1,135 @@
+import { AuthContextModule } from "@artsy/cohesion"
+import { Box, ResponsiveBox } from "@artsy/palette"
+import { MyCollectionGridItem_artwork } from "v2/__generated__/MyCollectionGridItem_artwork.graphql"
+import { useSystemContext } from "v2/System"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+import { userIsTeam } from "v2/Utils/user"
+import { cropped, resized } from "v2/Utils/resized"
+import { useHoverMetadata } from "v2/Components/Artwork/useHoverMetadata"
+import { DetailsFragmentContainer as Details } from "v2/Components/Artwork/Details"
+import { MagnifyImage } from "v2/Components/MagnifyImage"
+
+interface MyCollectionArtworkGridItemProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  artwork: MyCollectionGridItem_artwork
+  contextModule?: AuthContextModule
+  lazyLoad?: boolean
+  onClick?: () => void
+}
+
+export const MyCollectionArtworkGridItem: React.FC<MyCollectionArtworkGridItemProps> = ({
+  artwork,
+  lazyLoad = true,
+  contextModule,
+  onClick,
+  ...rest
+}) => {
+  const { user } = useSystemContext()
+  const isTeam = userIsTeam(user)
```

Comment:
nice hook 😄

Reply:
I just realised that I can right-click on an image when I'm logged in with my Artsy account :)

Reply:
The fact that we disable this bothers me to no end :)